locales/templates/app.pot

```diff
@@ -1965,11 +1964,11 @@ msgstr ""
 
 #. "I'm In!" is a button where users can press to join the reward program
 msgctxt "msrp_challenge_button_1"
-msgid "I’m In!"
+msgid "I'm In!"
```

Comment:
Not sure why this has been updated since it's supposed to be a curly apostrophe

Reply:
yeah, my bad, it should be curly apostrophe